internal/host/validator.go

```diff
@@ -808,7 +810,9 @@ func (v *validator) printSufficientPacketLossRequirementForRole(c *validationCon
 			return fmt.Sprintf("Error while attempting to validate packet loss validation: %s", err)
 		}
 		sort.Strings(hostNames)
-		return fmt.Sprintf("Packet loss percentage requirement of less or equals than %.2f%% not met for connectivity between %s and %s.", *c.clusterHostRequirements.Total.PacketLossPercentage, c.host.RequestedHostname, strings.Join(hostNames, ","))
+		return fmt.Sprintf("Packet loss percentage requirement of less or equals than %.2f%% not met for connectivity between %s and %s.", *c.clusterHostRequirements.Total.PacketLossPercentage, c.host.ID, strings.Join(hostNames, ","))
+	case ValidationPending:
```

Comment:
The validation function does not return pending for connectivity

Reply:
you're right. I mixed it up with #2251